### ISSUE TITLE
Publish Unresponsive

### ISSUE THREAD

**Issue by custom-jonathan** [urbit/urbit#2574]
**Describe the bug**
My group publish is unresponsive. Nobody in the group can publish a new note. My personal notebook won't add a new note, either.

**To Reproduce**
1. Go to "The New Era" add the title & body of the post.
2. Click on "Publish to the new era"
3. The indicator square spins in the upper corner, and I left it for 8 hours.

**Expected behavior**
Click "publish" and it updates

**Desktop (please complete the following information):**
 - OS: Digital Ocean Droplet Ubuntu
 - Browser Brave, Chrome, & Firefox
 - Base hash of your urbit ship. Run `[ [%base-hash 0v19.ibduh.62jie.3rk7j.ojsiq.4bmhv.m7cus.c8cu5.rhoq7.rs3pp.f0eri]`

**Additional context**
I've tried changing my `|sync` star this morning to see if that would kickstart my update.

**Comment by custom-jonathan** [comment]
I ran `|cancel` and `<<behn>>` started immediately. It's been going for an hour, and I tried to `ctl+c` to interrupt and nothing happened. Hit `ctl+z` twice and it dumped the core. Started the ship back up and `<<behn>>` started right after replay and I'm back to the `<<behn>>` spinner.

**Comment by custom-jonathan** [comment]
I got my ship workable by `ctl+z`ing a couple times to total ship shutdown. Then, start the ship back up and as soon at `<<behn>>` showed up I'd hit `ctl+c` and do it several times. Then it would eventually stop showing up, and I was able to turn off the autosync from my star, then ran `|cancel` which seemed to process what was hanging up. Then I turned sync back on and I eventually got the right OTA and the test notebook posts came through.
